spacemesh/v1/admin_types.proto

```diff
@@ -42,6 +42,11 @@ message Event {
     EventMalfeasance malfeasance = 15;
     EventPostServiceStarted post_service_started = 16;
     EventPostServiceStopped post_service_stopped = 17;
+    EventWaitingForPoETRegistrationWindow waiting_for_poet_registration_window = 18;
+    EventProofDownloadedFromPoet proof_downloaded_from_poet = 19;
+    EventRegisteredInPoet registered_in_poet = 20;
+    EventBestProofSelected best_proof_selected = 21;
+	EventWaitingForPoETRoundEnd waiting_for_poet_round_end = 22;
```

Comment:
NIT: Indentation is different from other lines 🙂

Reply:
I'm still fixing project settings and looks like i need to enable showing me whitechars :D
Fixed, thx!